src/Toolbar/Toolbar.example.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { Button } from 'antd';
```

Comment:
Could be changed to `SimpleButton` of `react-geo` in https://github.com/terrestris/react-geo/pull/1 is merged.

Reply:
Just merge it. I'll change this in #1 afterwards...